hack/e2e/environment.sh

```diff
@@ -21,6 +21,22 @@ function os_check() {
 export CAPI_VERSION=${CAPI_VERSION:-"v1beta1"}
 export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export NUM_NODES=${NUM_NODES:-"4"}
+export UPGRADE_TEST=${UPGRADE_TEST:-false}
+if $UPGRADE_TEST; then
+    export CAPI_VERSION="v1alpha4"
+    export CAPM3_VERSION="v1alpha5"
+fi
+
+# Override project infra vars that point to 
+# the current branch to build capm3 image and crds
+if [[ "${CAPM3_VERSION}" == "v1alpha5" ]]; then 
+    export CAPM3RELEASE="v0.5.4"
+    export CAPIRELEASE="v0.4.7"
+    export CAPM3BRANCH="release-0.5"
+    unset CAPM3_LOCAL_IMAGE
```

Comment:
What is the reason to unset this?

Reply:
This var is set in[ project infra](https://github.com/metal3-io/project-infra/blob/9470543342be88a38a94e120b81ee04c90a9576b/jenkins/scripts/integration_test_env.sh#L63)  to direct the capm3 location to tested_pr while we want in the upgrade to start with an old version cloned under go/src/..

Reply:
This is set to the same location as you are trying to set later, basically, you are unsetting/setting back to the same location IMO.